src/plan.rs

```diff
@@ -191,6 +193,52 @@ impl Recipe {
             Recipe::BluesteinsAlgorithm { len, .. } => *len,
         }
     }
+
+    pub fn cost(&self, repeats: usize) -> f32 {
+        let repeats_f = repeats as f32;
+        match self {
+            // TODO measure DFT
+            Recipe::Dft(len) => (50.0 * (*len as f32).powf(2.2)) * repeats_f,
+            Recipe::Radix4(len) => estimate_radix4_cost(*len, repeats),
+            Recipe::Butterfly1 => BUTTERFLY_COST_FACTOR * estimate_butterfly_cost_2(repeats), //TODO
```

Comment:
What's this TODO for?

Reply:
It's a reminder that I should add benches and fit the DFT as well. Or I just remove it from here, no point in estimating something that will never win the bidding anyway..

Reply:
True. Maybe the best thing to do is to remove the Dft recipe entirely. I know the "butterfly 0" case uses it, but we could have a "Butterfly0" recipe that returns a DFT(0) when constructed instead.